Create ui-c8y-1021-22-61-if-a-new-dashboard-is-added-the-first-navigation-does-not.md (#3152)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1021-22-61-if-a-new-dashboard-is-added-the-first-navigation-does-not.md

---
date: ""

> title: Fixed an issue with navigation not working when adding a new dashboard

Reviewer: ```suggestion
title: Fixed an issue with the navigation after adding a new dashboard
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YdSEScrEC
    label: Cockpit
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-62921
version: 1021.22.61
---

> If a new dashboard is added, the first navigation does not work #8496 [GRAFT][release/y2025] (#8640)

Reviewer 2: ```suggestion
After creating new dashboard, user was navigated to new dashboard but first navigation away from dashboard view was cancelled. It was fixed and navigation works.
```

> After creating new dashboard, user was navigated to new dashboard but first navigation away from dashboard view was cancelled. It was fixed and navigation works.

Reviewer: ```suggestion
After creating a new dashboard, when navigating to another menu item in the navigator, the navigation was cancelled. This issue has been fixed and the navigation works properly again.
```
